crates/xilem_masonry/src/view/flex.rs

```diff
@@ -16,6 +19,13 @@ pub fn flex<VT, Marker>(sequence: VT) -> Flex<VT, Marker> {
     }
 }
 
+pub fn flex_item<V>(view: V, params: impl Into<FlexParams>) -> FlexWrapper<V> {
```

Comment:
More of a UX thing and not entirely relevant to that PR, we could also achieve this via an extension trait. So that this is possible via the builder pattern (maybe also within `MasonryView`, to avoid extra traits? (see also e.g. [here](https://github.com/Philipp-M/trui/blob/dd24a2921eb0587dfffbbafea4a3e62b92f39c43/src/view.rs#L56)))